### DISCUSSION TITLE
Solicitação de email para o CDC da PyBR24

### DISCUSSION THREAD

**Discussion by AndressaLF** [apyb/comunidade#157] [Geral]
Gostaria de solicitar um email para ser usado para criação do formulário da PyBR 2024. @naanadr @lidymonteiro

**Comment by naanadr** [comment]
@lidymonteiro qual o seu e-mail?

**Reply by lidymonteiro** [reply]
contato.lidymonteiro@gmail.com

**Comment by naanadr** [comment]
@lidymonteiro informou que conseguiu acessar o e-mail.

O e-mail criado foi o `cdc@python.org.br`
